### ISSUE TITLE
Queue section auto-update issue

### ISSUE THREAD

**Issue by pasan93** [chr24x7/chr247.com#14]
Queue section is not updating automatically. 
The user has to refresh the page to view the newly added patients.

**Comment by francisguchie** [comment]
Was this issue solved?

**Comment by IMS94** [comment]
@francisguchie No. I couldn't have a look at this and I haven't tried to reproduce this as well.

**Comment by francisguchie** [comment]
@IMS94 

thank you i will look into this once i interact with the system more
